c2rust-transpile/src/translator/assembly.rs

```diff
@@ -101,7 +131,24 @@ fn parse_constraints(mut constraints: &str) ->
         "r" => {
             constraints = "reg".into();
         }
-        _ => {},
+        _ => {
+            let is_explicit_reg = constraints.starts_with('"');
+            let is_tied = !constraints.contains(|c: char| !c.is_ascii_digit());
+
+            if !(is_explicit_reg || is_tied) {
+                // Attempt to parse machine-specific constraints
+                if let Some((machine_constraints, is_mem)) =
+                translate_machine_constraint(&*constraints, arch) {
+                    constraints = machine_constraints.into();
+                    mem_only = is_mem;
+                } else {
+                    warn!("Did not recognize inline asm constraint: {}\n\
+                    It is likely that this will cause compilation errors or \
+                    incorrect semantics in the translated program; please
```

Comment:
Do you need the escapes? If so it's missing on this line.

Reply:
The escape prevents the leading whitespace on the next line from being part of the string, so it is needed. Will fix.